apps/ideaspace/src/components/modules/IdeasBetaFeedbackModal/IdeasBetaFeedbackModal.js

```diff
@@ -1,36 +1,39 @@
 import Modal from 'react-modal';
 import { useState, useEffect } from 'react';
-import './IdeasBetaFeedbackModal.css';
+import style from './IdeasBetaFeedbackModal.module.css';
 import messageIcon from './message-square.svg';
 import modalImage from './modal-img.png';
 
+
 const formLink =
   'https://docs.google.com/forms/d/e/1FAIpQLSdkipdKLxq4dGQf3dX5Sdfb_XNoPhhGsNxPXNcNNSMzhIrYxg/viewform';
 
 const ModalCustomStyles = {
   overlay: {
-    position: 'fixed',
-    top: 0,
-    left: 0,
-    right: 0,
-    bottom: 0,
-    backgroundColor: 'transparent',
-    zIndex: 11,
+    // position: 'fixed',
+    // top: 0,
+    // left: 0,
+    // right: 0,
+    // bottom: 0,
+    // backgroundColor: 'transparent',
+    // zIndex: 11,
   },
   content: {
-    fontFamily: 'Nunito Sans',
-    top: 'auto',
-    left: 'auto',
-    right: '2rem',
-    bottom: '2rem',
-    maxWidth: '320px',
-    paddingTop: '80px',
-    paddingBottom: '0',
-    textAlign: 'center',
-    height: '550px',
-    border: 'none',
-    borderRadius: '16px',
-    boxShadow: '0px 0px 20px rgba(0, 0, 0, 0.1)',
+    // fontFamily: 'Nunito Sans',
+    // top: 'auto',
+    // left: 'auto',
+    // right: '2rem',
+    // bottom: '2rem',
+    // maxWidth: '320px',
+    // paddingTop: '80px',
+    // paddingBottom: '0',
+    // textAlign: 'center',
+    // height: '550px',
+    // border: 'none',
+    // borderRadius: '16px',
+    // boxShadow: '0px 0px 20px rgba(0, 0, 0, 0.1)',
+    // transition: '0.4s all ease-in-out',
+    // transform: 'translateX(200%)',
```

Comment:
can we delete these lines instead of commenting?

Reply:
@dbradham these changes are from your end, are you sure your team didn't comment this just for a bit then uncomment?

Reply:
nvm, I wonder what was the reason for doing this but yeah sure I think it's safe to be deleted!